src/WebJobs.Script/Description/DotNet/Compilation/Raw/RawAssemblyCompilation.cs

```diff
@@ -53,7 +53,7 @@ public FunctionSignature GetEntryPointSignature(IFunctionEntryPointResolver entr
                 throw new InvalidOperationException($"The function type name '{typeName}' is invalid.");
             }
 
-            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Static | BindingFlags.Public);
+            MethodInfo method = functionType.GetMethod(methodName, BindingFlags.Public | BindingFlags.Static | BindingFlags.Instance | BindingFlags.DeclaredOnly);
```

Comment:
Using the same flags that the core SDK uses (https://github.com/Azure/azure-webjobs-sdk/blob/v2.x/src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs#L27)

Reply:
This doesn't just apply to direct load, but to the raw assembly model as well. We need to ensure we have a test that validates that those classes will be properly instantiated to call instance methods. Did you have a chance to test that?